main.tf

```diff
@@ -10,4 +10,11 @@ resource "aws_lb" "test" {
   tags = {
     Environment = "production"
   }
+
+  access_logs {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure deletion protection is enabled for AWS LB (Load Balancer)**
 
**Vulnerability**
AWS Classic load balancer has deletion protection disabled. This may lead to accidental deletion of the load balancer.
 
**Remediation**
Enable deletion protection for classic load balancer. This will prevent accidental deletion of the resource.

source: accurics-stagetwo